RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ImportMnemonicControllingAccounts+View.swift

```diff
@@ -54,7 +54,6 @@ extension ImportMnemonicControllingAccounts {
 							store: store.scope(state: \.entities, action: { .child(.entities($0)) })
 						)
 					}
-					.fixedSize(horizontal: false, vertical: true) // prevent ScrollView from growing vertically if not needed.
```

Comment:
What I really wanted to do is some SwiftUI version of UIKits "content hugging priority". But seems not possible in SwiftUI? It only has [`layoutpriority(_:)`](https://developer.apple.com/documentation/swiftui/view/layoutpriority(_:)) which seems akin to "Compression Resistance".